src/app/pages/products-list/products-list.component.ts

```diff
@@ -1,9 +1,31 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {productsMock} from '../../shared/products/products.mock';
+import {Product} from '../../shared/products/product.interface';
 
 @Component({
     selector: 'app-products-list',
     templateUrl: './products-list.component.html',
     styleUrls: ['./products-list.component.css'],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ProductsListComponent {}
+export class ProductsListComponent {
+    readonly productData: Product;
+    readonly cart: Array<Product['_id']> = [];
+
+    constructor() {
+        this.productData = this.randomProduct;
+    }
```

Comment:
Да, хорошо, что инициализацию производим в конструкторе, но можно было бы это сделать через инициализацию свойства
```suggestion
    readonly productData = this.randomProduct;
```

Reply:
Не уверен был как лучше сделать. Исправил &#128532;

Reply:
По возможности лучше пользоваться инициализацией через свойства, это читается проще, понятней, и занимает меньше места